### ISSUE TITLE
Track Glyphs 2.5 features

### ISSUE THREAD

**Issue by madig** [googlei18n/glyphsLib#349]
Sift through https://glyphsapp.com/blog/new-features-in-glyphs-2-5 to see what needs to be implemented.

**Comment by madig** [comment]
Thanks! Can you detail what other changes would need to be added?

**Comment by schriftgestalt** [comment]
From the top of my head, there are two "structural" changes. Double unicodes and the origin/target.. 'Points' of hints can be two, three or four numbers long. That is needed for TT-hints on intersection points.

**Comment by schriftgestalt** [comment]
I don’t think that there are more things. The Axes parameter is handled already.

**Comment by schriftgestalt** [comment]
can this be closed?

**Comment by madig** [comment]
Hinting still crashes glyphsLib.
